google/cloud/aiplatform/jobs.py

```diff
@@ -1913,3 +1918,522 @@ def run(
     def trials(self) -> List[gca_study_compat.Trial]:
         self._assert_gca_resource_is_available()
         return list(self._gca_resource.trials)
+
+
+class ModelDeploymentMonitoringJob(_Job):
+    """Vertex AI Model Deployment Monitoring Job.
+
+    This class should be used in conjunction with the Endpoint class
+    in order to configure model monitoring for deployed models.
+    """
+
+    _resource_noun = "modelDeploymentMonitoringJobs"
+    _getter_method = "get_model_deployment_monitoring_job"
+    _list_method = "list_model_deployment_monitoring_jobs"
+    _cancel_method = None
+    _delete_method = "delete_model_deployment_monitoring_job"
+    _job_type = "model-deployment-monitoring"
+    _parse_resource_name_method = "parse_model_deployment_monitoring_job_path"
+    _format_resource_name_method = "model_deployment_monitoring_job_path"
+
+    def __init__(
+        self,
+        model_deployment_monitoring_job_name: str,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ):
+        """Initializer for ModelDeploymentMonitoringJob
+
+        Args:
+            model_deployment_monitoring_job_name (str):
+                Required. A fully-qualified ModelDeploymentMonitoringJob resource name or ID.
+                Example: "projects/.../locations/.../modelDeploymentMonitoringJobs/456" or
+                "456" when project and location are initialized or passed.
+            project: Optional[str] = None,
```

Comment:
Docstring types should not include Optional or the default argument. Please update throughout.